### DISCUSSION TITLE
Reactive features like Getx?

### DISCUSSION THREAD

**Discussion by Im-PJ** [nylo-core/nylo#92] [Ideas]
For a beginners app, most of the scenarios are covered by Nylo. However, I miss reactive features like using observables to update widgets. Is there any plans for reactive extensions in Nylo or would you suggest using getx/mobx alongside nylo?

**Comment by agordn52** [comment]
Hi @Im-PJ,

Here's a new video on the YouTube channel that explains how you can use state management https://www.youtube.com/watch?v=X5EVh1KooFk 

Hope that answers your question, thanks
